src/components/EventDashboard/ActionConfirmationDialog.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {
+  makeStyles,
+  Button,
+  List,
+  ListItem,
+  ListItemText,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+} from '@material-ui/core';
+import { Colours } from '../../styles/Constants';
+import { TabOptions } from './EventDashboardPage';
+import { ResourceData, HospitalData, AmbulanceData } from './ResourceTabPanel';
+
+interface ActionConfirmationDialogProps {
+  open: boolean;
+  type: TabOptions.Hospital | TabOptions.Ambulance;
+  handleClose: () => void;
+  handleAdd: () => void;
+  handleDelete: () => void;
+  includeOrExclude: 'include' | 'exclude' | null;
```

Comment:
would this be more performant than string comparisons?
```suggestion
  isInclude: boolean;
```

Reply:
I don't think it makes a difference. Also would prefer having three distinct states because `false` and `null` can become kind of ambiguous.

Reply:
technically the boolean option does give you three distinct states (true, false, null), but you're right - shouldn't really do much in terms of perf anyways